python/graphstorm/graphpartitioning/array_readwriter/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
as we discussed, the code doesn't belong to the library.

Reply:
As I agreed with you earlier... moved this code to the test folder. Since it is expected to be used ONLY for testing purposes. The application needs only reading of the files, application never writes files to the disk. Just for reading files from the disk, we can use a much simplified function to read files from the disk.